scripts/build_posix.sh

```diff
@@ -46,6 +47,7 @@ fpm_build()
 	fpm -s dir \
 	    -f \
 	    -t $1 \
+		--description $DESC \
```

Comment:
```suggestion
	    --description $DESC \
```

Reply:
Should I remove two spaces or add two spaces